src/app/organisms/room/RoomViewHeader.jsx

```diff
@@ -95,9 +94,7 @@ function RoomViewHeader({ roomId }) {
         <RawIcon src={ChevronBottomIC} />
       </button>
       <IconButton onClick={() => toggleRoomSettings(tabText.SEARCH)} tooltip="Search" src={SearchIC} />
-      <div className="room-people-drawer-toggle">
-        <IconButton onClick={togglePeopleDrawer} tooltip="People" src={UserIC} />
-      </div>
+      <IconButton className="room-people-drawer__toggle" onClick={togglePeopleDrawer} tooltip="People" src={UserIC} />
```

Comment:
className doesn't follow BEM. it will be like room-header__drawer-btn. ([block]__[element]--[modifier]) block is room-header, element is drawer-btn